types/three/three-mapcontrols.d.ts

```diff
@@ -0,0 +1,70 @@
+import {Camera, MOUSE, Object3D, Vector3} from 'three'
+
+export class MapControls {
```

Comment:
Seem to be missing 3 members, `target0`, `position0`, and `zoom0` https://github.com/mrdoob/three.js/blob/dev/examples/js/controls/MapControls.js#L85

They're in the "public" section of the code, not sure how well documented they are.